Watchman.Discord/Areas/Reactions/Contexts/ReactionContext.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Discord;
+using Discord.WebSocket;
+
+namespace Watchman.Discord.Areas.Reactions.Contexts
+{
+    public class ReactionContext
+    {
+        public ISocketMessageChannel ChannelContext { get; }
```

Comment:
tu nie powinno być obiektów z Discord.NET

Reply:
A jakbym zrobił tak jak mówi Mateusz, że dać ReactionContext do DiscordFramework to będzie ok że te obiekty są? Czy obiekty z Discord.NET zamienić na coś?

Reply:
przenieś do frameworka, jak i również zastąp elementy Discord.NET

Reply:
Jeśli o IEmote to jest łatwa sprawa bo tam jest tylko informacja o emotce, ale jak zastąpić IUser i ISocketMessageChannel? Stworzyć po prostu klasy, które będą miały te same properties a metody tych nowo powstałych klas mają wywoływać metody IUser i ISocketMessageChannel? Nic innego mi do głowy nie przychodzi.

Reply:
spójrz jak działają obecne contexty
np UserContext, ChannelContext itd